src/NzbDrone.Core/Notifications/Telegram/TelegramSettings.cs

```diff
@@ -1,10 +1,18 @@
+using System.ComponentModel;
 using FluentValidation;
 using NzbDrone.Core.Annotations;
 using NzbDrone.Core.ThingiProvider;
 using NzbDrone.Core.Validation;
-
 namespace NzbDrone.Core.Notifications.Telegram
 {
+    public enum MetadataLinkType
```

Comment:
Move below the class

Reply:
Add `None =0` as the first enum value to allow the single enum to control whether it's sent or not.